src/libs/sfdk/sdk.cpp

```diff
@@ -117,6 +118,21 @@ Sdk *Sdk::instance()
     return s_instance;
 }
 
+QString Sdk::osVariant(const char *separator)
+{
+    return tr(Constants::VARIANT_NAME) + separator + tr(Constants::OS_NAME);
+}
+
+QString Sdk::sdkVariant(const char *separator)
+{
+    return tr(Constants::VARIANT_NAME) + separator + tr(Constants::SDK_NAME);
+}
+
+QString Sdk::ideVariant(const char *separator)
+{
+    return tr(Constants::VARIANT_NAME) + separator + tr(Constants::IDE_NAME);
```

Comment:
OS_NAME, SDK_NAME and IDE_NAME feel wrong. Maybe we could use the strings literally here instead?